diamond_agent/tasks.py

```diff
@@ -220,8 +220,21 @@ def config_handlers(ctx, handlers, config_path, handlers_path):
     """
     if handlers is None:
         handlers = copy_objects.deepcopy(DEFAULT_HANDLERS)
-        handlers['cloudify_handler.cloudify.CloudifyHandler']['config'][
-            'server'] = get_manager_ip()
+
+        cloudify_agent = ctx.bootstrap_context.cloudify_agent
```

Comment:
In this case, should I expect this to raise an exception or should I expect that cloudify_agent is set to None (or something else)?